configure.ac

```diff
@@ -39,7 +39,7 @@ AC_CONFIG_MACRO_DIR([tools])
 AM_INIT_AUTOMAKE([1.11 foreign dist-xz no-dist-gzip subdir-objects])
 # we want tar-ustar to avoid introducing extra metadata (ctime, atime) which
 # only adds useless non-determinism to the result.  we also want to sort.
-am__tar='tar --format=ustar --sort=name -chf - "$$tardir"'
+am__tar='tar --format=ustar --sort=name --owner=root:0 --group=root:0 -chf - "$$tardir"'
```

Comment:
This is a fixup for "build: change timestamp mangling approach", so it'd be nicer to squash it instead of landing a broken commit.